client/extensions/woocommerce/components/image-thumb/index.js

```diff
@@ -0,0 +1,53 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+const ImageThumb = ( { width, height, src, alt, ...props } ) => {
+	const style = {
+		width,
+		height,
+		minHeight: height,
+	};
+
+	const imageClasses = classNames( 'image-thumb', {
+		hasImage: src,
+	} );
+	const placeholderClasses = classNames( 'image-thumb__placeholder' );
```

Comment:
This doesn't need the `classNames` call - could you just put this directly on the div?